knx.py

```diff
@@ -14,6 +14,8 @@
 KNXWRITE = 0x80
 KNXREAD = 0x00
 
+DT_BIT = '1.*'
```

Comment:
The Prefixes are standardized by KNX. The KNX Specifications contain a list of all data types defined.